RELEASE_NOTES.txt

```diff
@@ -1,3 +1,27 @@
+MUSIC v2.0.0:
+- Scrips pipelines in Python
```

Comment:
Scripts

Reply:
Actually I would just put Python pipelines (as I said before I don't like the term script)